packages/simple-markdown/README.md

```diff
@@ -0,0 +1,457 @@
+simple-markdown
+===============
+
+simple-markdown is a markdown-like parser designed for simplicity
+and extensibility.
+
+[Change log](https://github.com/Khan/simple-markdown/releases)
+
+Philosophy
+----------
+
+Most markdown-like parsers aim for [speed][marked] or
+[edge case handling][CommonMark].
+simple-markdown aims for extensibility and simplicity.
+
+[marked]: https://github.com/chjj/marked
+[CommonMark]: https://github.com/jgm/CommonMark
+
+What does this mean?
+Many websites using markdown-like languages have custom extensions,
+such as `@`mentions or issue number linking. Unfortunately, most
+markdown-like parsers don't allow extension without
+forking, and can be difficult to modify even when forked.
+simple-markdown is designed to allow simple addition of
+custom extensions without needing to be forked.
+
+At Khan Academy, we use simple-markdown to format
+over half of our math exercises, because we need
+[markdown extensions][PerseusMarkdown] for math text and
+interactive widgets.
+
+[PerseusMarkdown]: https://github.com/Khan/perseus/blob/master/src/perseus-markdown.jsx
+
+simple-markdown is [MIT licensed][LICENSE].
+
+[LICENSE]: https://github.com/Khan/simple-markdown/blob/master/LICENSE
+
+Getting started
+---------------
+
+First, let's parse and output some generic markdown using
+simple-markdown.
+
+If you want to run these examples in
+node, you should run `npm install` in the simple-markdown
+folder or `npm install simple-markdown` in your project's
+folder. Then you can acquire the `SimpleMarkdown` variable
+with:
+
+```javascript
+    var SimpleMarkdown = require("simple-markdown");
```

Comment:
Can you update these code samples to use `import SimpleMarkdown from '@khanacademy/simple-markdown'`?